src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests4.cs

```diff
@@ -2052,7 +2052,7 @@ public class SwitchExpressionException : InvalidOperationException
                 //             _ = (1, 2) switch { (3, 4) => 1 };
                 Diagnostic(ErrorCode.WRN_SwitchExpressionNotExhaustive, "switch").WithLocation(8, 24)
                 );
-            CompileAndVerify(compilation, expectedOutput: "SwitchExpressionException()");
+            CompileAndVerify(compilation, expectedOutput: "SwitchExpressionException((1, 2))");
```

Comment:
Please add a test with missing `ValueTuple` type.
Maybe testing a long tuple could be good too.